demo/src/main/res/values/strings.xml

```diff
@@ -17,6 +17,7 @@
     <string name="navigation_item_components">Components</string>
     <string name="navigation_item_modules">Modules</string>
     <string name="navigation_item_about">About</string>
+    <string name="navigation_item_search"></string>
```

Comment:
As suggested by Android Studio, you can add `tools:ignore="ExtraTranslation"` cause the string is empty